### ISSUE TITLE
Failed to update NixOS Options

### ISSUE THREAD

**Issue by austinbutler** [mlvzk/manix#19]
```
Building Nixpkgs comments cache...
Building Home Manager Options cache...
Building NixOS Options cache...
Failed to update NixOS Options

Caused by:
    0: Failed to perform IO on a cache file
    1: No such file or directory (os error 2)
```

manix 0.6.2
macOS

**Comment by mlvzk** [comment]
What channel are you on?

**Comment by austinbutler** [comment]
```
❯ nix-channel --list
home-manager https://github.com/rycee/home-manager/archive/master.tar.gz
nixpkgs https://nixos.org/channels/nixpkgs-unstable
```

**Comment by nrdxp** [comment]
This is still an issue. Any ideas?

**Comment by mlvzk** [comment]
This should be fixed on version 0.6.3.

Sorry for taking so long, I only now updated to 20.09 from 20.03.
